doc/factors.md

```diff
@@ -0,0 +1,192 @@
+# Using marian with factors
```

Comment:
The documentation looks very good, it's clear and concise. Thanks.

I notice that it describes training, but for completeness, it would be nice to have basic one-line examples with training and decoding commands. Maybe in a separate short section at the end. People will surely ask about this.

Reply:
Even being a older message, just replying here that this was already added to the [documentation](https://github.com/marian-cef/marian-dev/blob/master/doc/factors.md#examples), to mark it as done once you are closing the discussion threads.